### DISCUSSION TITLE
python on IOS

### DISCUSSION THREAD

**Discussion by yoelnavon** [PythonFreeCourse/Notebooks#55] [Q&A]
Hi ,how to install python on IOS?

**Comment by yammesicka** [comment]
The notebooks aren't supported in mobile phone environments.
Please use a desktop setup instead.

**Comment by yoelnavon** [comment]
hi,
i'm work in desktop

‫בתאריך יום ב׳, 1 בנוב׳ 2021 ב-17:35 מאת ‪Yam Mesicka‬‏ <‪
***@***.***‬‏>:‬

**Comment by yoelnavon** [comment]
macbook

‫בתאריך יום ב׳, 1 בנוב׳ 2021 ב-17:36 מאת ‪yoel navon‬‏ ***@***.***
‬‏>:‬

**Comment by yoelnavon** [comment]
בנוסף הייתי שמח לדעת מדוע אני רואה ככה את מסך הלמידה?
יש פתרון לזה?
תודה רבה (:

‫בתאריך יום ב׳, 1 בנוב׳ 2021 ב-17:38 מאת ‪yoel navon‬‏ ***@***.***
‬‏>:‬

**Reply by yammesicka** [reply]
עקבת אחרי המדריך הכתוב בעמוד הזה?
https://github.com/PythonFreeCourse/Notebooks

**Comment by HananelLivneh** [comment]
The Notebooks page of this git has a link to a video on how to install using Anaconda.
You can also find information on the direct installation of Python here: https://docs.python.org/3/using/mac.html, yet I recommend you follow the instructions of installing using Anaconda: https://docs.anaconda.com/anaconda/install/mac-os/
Information regarding Jupiter can be found here: https://jupyter.org/install

On a side note - if you plan to learn a coding language, you will need to learn how to solve such problems by googling the question. Get used to googling and searching for answers as quickly as possible and your learning experience will improve.

**Comment by yoelnavon** [comment]
[image: image.png]

עקבתי אחרי המדריך, אבל הוא לא נתן לי פתרון לmacbook...
האם יש לכם מדריך איך להעלות את הקבצים בתוך הjupyter?

‫בתאריך יום ב׳, 1 בנוב׳ 2021 ב-18:35 מאת ‪Yam Mesicka‬‏ <‪
***@***.***‬‏>:‬
